addons/html_editor/static/src/others/embedded_components/plugins/video_plugin/video_plugin.js

```diff
@@ -1,73 +1,47 @@
 import { Plugin } from "@html_editor/plugin";
 import { _t } from "@web/core/l10n/translation";
-import { VideoSelectorDialog } from "@html_editor/others/embedded_components/plugins/video_plugin/video_selector_dialog/video_selector_dialog";
-import { renderToElement } from "@web/core/utils/render";
 import { isHtmlContentSupported } from "@html_editor/core/selection_plugin";
+import { EmbeddedVideoSelector } from "./video_selector_dialog/embedded_video_selector";
 
 export class VideoPlugin extends Plugin {
     static id = "video";
-    static dependencies = ["embeddedComponents", "dom", "selection", "link", "history"];
+    static dependencies = ["embeddedComponents", "media"];
+    static defaultConfig = {
+        allowEmbeddedVideo: true,
+    };
+    // Only include the embedded video selector tab in the media dialog if it
+    // doesn't already have the video tab.
+    shouldIncludeEmbeddedVideoSelector =
+        !this.config.allowMediaDialogVideo && this.config.allowEmbeddedVideo;
```

Comment:
Now that we're moving the "Video Link" command's dialog into the Media Dialog... 
I guess "allowMediaDialogVideo" means "allow inserting video directly as an iframe in the editable", as opposed to an embedded component.
We should probably rename this option....

Reply:
I'm not sure having two different options still makes sense anymore, in fact. I think it could be `config.allowVideo` and then use the existing `config.embeddedComponents` to infer which type of video we can use.